app/helpers/local_catalog_helper.rb

```diff
@@ -90,10 +90,13 @@ def jhu_link_to_document(doc, label, opts)
   # guess of whether the 856 links are for online access or not -- just
   # based on whether it's marked "Online" in format facet, at present.
   def related_links_type(document)
-    if document["format"] && document["format"].include?("Online")
-      :online
-    else
-      :related
+    case document["format"]
```

Comment:
@little9 I think there might be an issue here. document["format"] is going to be an array and because of that, I think everything is just falling through to the "else" case. I think there is some more work to be done around related links, but one of the potential issues I can see here is that related links could be available in more than just Print and Manuscripts. This is also an issue in production, but I have a ticket to work on that so I don't know that we need to fix it here.

Reply:
@fackrellj I changed that method and added some extra tests for its behavior. This record looks on on `-test` for me now: 

https://catalyst-test.library.jhu.edu/catalog/bib_4910858

The new logic (until we determine all the possible formats with related links) will check for `Print` in the format array. If it's not there it will default to `Online`.